duckduckgo/BrowserServicesKit
# Referrer Trimming

Please review the release process for BrowserServicesKit [here](https://app.asana.com/0/1200194497630846/1200837094583426).

**Required**:

Task/Issue URL: https://app.asana.com/0/72649045549333/1200810841420631/f
iOS PR: https://github.com/duckduckgo/iOS/pull/1272
macOS PR: https://github.com/more-duckduckgo-org/macos-browser/pull/682/
What kind of version bump will this require?: Minor (API is changed but adoption is not required)

**Optional**:

Tech Design URL: https://app.asana.com/0/481882893211075/1202694361990652/f
CC: @bwaresiak 

**Description**:
This PR adds support for referrer header trimming



**Steps to test this PR**:
1. Visit https://privacy-test-pages.glitch.me/privacy-protections/referrer-trimming/
3. Click "Start Test"
4. Expand the results
5. All 3p tests should be trimmed to the hostname "https://privacy-test-pages.glitch.me/"
6. Do some general smoke testing especially clicking links from one domain to another and ensure functionality seems correct.



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
